backend/api/serializers.py

```diff
@@ -14,9 +14,9 @@ class Meta:
         fields = '__all__'
 
 
-class PlaceSerializer(ModelSerializer):
+class StoreSerializer(ModelSerializer):
     class Meta:
-        model = Place
+        model = Store
         fields = '__all__'
```

Comment:
Why is it the store instead of the source? In generell I thought the file should contain not only FiBoUser but also LiteUser and PremiumUser. The same I expected for the SourceSpecializations. (probably I got unsufficient knowledge about it :D)

Reply:
Ah, I see. This is true, I guess that this will automatically be the case when all tests are passing again.